api/models.py

```diff
@@ -185,3 +185,29 @@ class SecurityUser(User):
 
     class Meta:
         verbose_name = "Security User"
+
+
+class AssetLog(models.Model):
+    """Stores checkin/Checkout asset logs"""
+    CHECKIN = "Checkin"
+    CHECKOUT = "Checkout"
+
+    option = (
+        (CHECKIN, "Checkin"),
+        (CHECKOUT, "Checkout"),
+    )
+    item = models.ForeignKey(Item,
+                             null=False,
+                             on_delete=models.PROTECT)
+    checked_by = models.ForeignKey(SecurityUser,
+                                   blank=True,
+                                   default=1,
```

Comment:
we're not sure there'll always be a security user with id 1, so we shouldn't set a default here. 
also can we have this field take in the currently logged in user by default?